src/lib.rs

```diff
@@ -8,163 +8,154 @@
  * except according to those terms.
  */
 
-extern crate cargo;
+extern crate cargo_metadata;
 extern crate time;
+extern crate toml;
 #[macro_use]
 extern crate structopt;
+#[macro_use]
+extern crate failure;
 
-use cargo::core::registry::PackageRegistry;
-use cargo::core::resolver::Method;
-use cargo::core::{Package, PackageSet, Resolve, Workspace};
-use cargo::ops;
-use cargo::util::{important_paths, CargoResult, CargoResultExt};
-use cargo::{CliResult, Config};
+pub use failure::Error; // re-exported to main
+use std::env;
+use std::fs::File;
 use std::fs::OpenOptions;
+use std::io::prelude::*;
 use std::io::Write;
+use std::path::Path;
 use std::path::PathBuf;
+use toml::Value;
 
 #[derive(Debug, StructOpt)]
 pub enum Command {
     #[structopt(name = "ebuild")]
     /// Build an ebuild file from a cargo project
-    Ebuild,
+    Ebuild {
+        #[structopt(long = "ebuild-path", short = "o")]
+        ebuild_path: Option<String>,
+        #[structopt(long = "manifest-path", short = "m")]
+        manifest_path: Option<String>,
+    },
 }
 
 /// Parse cli commands
-pub fn run(cmd: Option<Command>, verbose: u32, quiet: bool) -> CliResult {
```

Comment:
So you remove the verbose and quiet flags after making the prior commit mostly about them?

Reply:
the prior commit had nothing to do with verbosity and quiet